demos/notebooks/2_pydough_operations.ipynb

```diff
@@ -84,9 +84,9 @@
    "id": "a25a2965-4f88-4626-b326-caf931fdba9c",
```

Comment:
<p>Anything that can be an expression used in a&nbsp;<code><em>CALCULATE</em></code>&nbsp;or a&nbsp;<code>WHERE.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bodo-ai/PyDough/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>